kernel/src/engine/arrow_expression/evaluate_expression.rs

```diff
@@ -263,5 +265,6 @@ pub fn evaluate_predicate(
                 .reduce(|l, r| Ok(reducer(&l?, &r?)?))
                 .unwrap_or_else(|| Ok(BooleanArray::from(vec![default; batch.num_rows()])))
         }
+        Opaque(_) | Unknown(_) => todo!(),
```

Comment:
should we also make this just an `Err(Error::unsupported(...))` for now so we don't panic?

Reply:
hahaha happens!!